ibtk/src/lagrangian/FEProjector.cpp

```diff
@@ -86,17 +88,21 @@ get_dirichlet_bdry_ids(const std::vector<boundary_id_type>& bdry_ids)
 /////////////////////////////// PUBLIC ///////////////////////////////////////
 
 FEProjector::FEProjector(EquationSystems* equation_systems, const bool enable_logging)
-    : d_fe_data(new FEData("FEProjector", *equation_systems, /*register_for_restart*/ false)),
-      d_enable_logging(enable_logging)
+    : d_fe_data(new FEData("FEProjector", /*register_for_restart*/ false)), d_enable_logging(enable_logging)
```

Comment:
In #1104 we changed `FEData` to take an `EquationSystems` in its constructor - that's causing the build failure.

Reply:
Fixed I think.